packages/gatsby-transformer-remark/src/gatsby-node.js

```diff
@@ -9,15 +9,9 @@ exports.setFieldsOnGraphQLNodeType = require(`./extend-node-type`)
 
 exports.pluginOptionsSchema = function ({ Joi }) {
   return Joi.object({
-    commonmark: Joi.boolean().description(
-      `Activates CommonMark mode (default: true)`
-    ),
```

Comment:
We shouldn't remove those options in a minor/patch version. Let's allow any value for now and schedule removal in the next major:

```
    // Options `commonmark` and pedantic have no effect since gatsby-transformer-remark@4.0.0
    // TODO: remove in v5
    commonmark: Joi.any(),
    pedantic: Joi.any(),
```

Reply:
Fixed.